Formula/transmission-cli.rb

```diff
@@ -24,31 +20,42 @@ class TransmissionCli < Formula
     sha256 x86_64_linux:   "178d05964e9efd8d4541cf5589f4772ccdc59b8de83158f96b8ad7ffeff6b8d2"
   end
 
+  depends_on "cmake" => :build
+  depends_on "gettext" => :build
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
   depends_on "libevent"
   depends_on "openssl@1.1"
 
   uses_from_macos "curl"
   uses_from_macos "zlib"
 
+  on_linux do
+    depends_on "python@3.11" => :build
+  end
+
   def install
+    args = %W[
+      -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Comment:
Why?

Reply:
I'm following the instructions here: https://github.com/transmission/transmission/blob/main/README.md#building-a-transmission-release-from-the-command-line

This uses `RelWithDebInfo`

Reply:
That's useful if someone is making their own release to debug something. But for actual releases we want to let brew set this.